doc/Language/101-basics.pod6

```diff
@@ -342,6 +356,11 @@ C<sort> sorts in ascending order, from smallest to largest. This is the
 opposite of the desired order. Therefore, the code calls the C<.reverse> method
 on the result of the second sort, and stores the final list in C<@sorted>.
 
+X<|say (Basics)>
```

Comment:
Again, why index this? Or, for that matter, mostly anything here if it's not explained anywhere? It's supposed to be a tutorial, if we add all kinds of entries we might end up with too many of them...

Reply:
[This was already indexed](https://github.com/Raku/doc/blob/master/doc/Language/101-basics.pod6#L353).

Reply:
Yep, as I say above, maybe too many things were indexed here. And if it was, what did you do precisely here? Change the anchor?